server/routes/stripe.js

```diff
@@ -73,6 +74,13 @@ router.post('/webhook', bodyParser.raw({ type: 'application/json' }), async (req
           agency: event.data.object.metadata.agencyName,
         });
 
+        await createNewDonation({
```

Comment:
this doesnt follow the rest of the project i think, everywhere else is repository.function()

while this might be shorter I think its better to keep the same style everywhere.

Reply:
also since you dont need a response I dont think you need an await here